lib/media-queries.js

```diff
@@ -0,0 +1,43 @@
+
+var _ = require('lodash')
+var postcss = require('postcss')
+var rules = require('./rules')
+var selectors = require('./selectors')
+var declarations = require('./declarations')
+
+module.exports = function (root, opts) {
+
+  var result = {
+    total: 0,
+    unique: 0,
+    values: [],
+    contents: [] // Not sure what to name this
```

Comment:
I wonder if this should be keyed off the values? That way the `contents` are associated with the media query value.

```js
var result = {
  total: 0,
  unique: 0,
  values: [],
  statsByValue: {
    '(min-width: 40em)': {}
  }
}
```